ex1.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
definirei tutte le funzioni prima di chiamarle, mettendo le chiamate alle funzioni in un main()